src/lib/core/rxjs/rx-chain.spec.ts

```diff
@@ -1,41 +0,0 @@
-import {Observable} from 'rxjs/Observable';
```

Comment:
It seems like this removes the `rx-chain` tests completely.

Reply:
They were already copied to `cdk` earlier but not removed from `lib`